samples/drivers/spi_flash_at45/sample.yaml

```diff
@@ -3,4 +3,4 @@ sample:
 tests:
   sample.drivers.spi.flash.at45:
     tags: spi flash
-    platform_whitelist: nrf9160_pca10090
+    platform_whitelist: nrf9160dk_nrf9160
```

Comment:
I think this sample will fail with TIMEOUT when run by sanitycheck even with correct hardware being provided as the sample does not provide a way to verify its correctness (if `build_only: true` is not used). (You can look at #26187 for an example of how this can be solved)